cnames_active.js

```diff
@@ -2696,6 +2696,7 @@ var cnames_active = {
   "workers": "worker-tools.github.io/workers.js.org",
   "worksnake": "worksnake.github.io",
   "wowufoundme": "wowufoundme.github.io",
+  "wpwebpack": "https://oblakstudio.github.io/wpwebpack",
```

Comment:
```suggestion
  "wpwebpack": "oblakstudio.github.io/wpwebpack",
```
Please remove the https://